### PR TITLE
Remove bintray in actions

### PR DESCRIPTION
**Pull request by skonto**
- removes bintray deps in github actions
- removes some other references to bintray
- makes actions script more generic so it can run from any dir locally as well.
It helps when a contributor works locally before issuing a PR.

Tested by removing all ivy and maven caches.

### CODE REVIEW

File: core/build.sbt [hunk 1]
```diff
@@ -452,7 +449,7 @@ def cloudflowModule(moduleID: String): Project = {
 lazy val commonSettings = Seq(
   organization := "com.lightbend.cloudflow",
   bintrayOrganization := Some("lightbend"),
-  bintrayRepository := altBintrayRepository.value.getOrElse("cloudflow"),
+  bintrayRepository := "cloudflow",
```
Comment (skonto) [inline core/build.sbt]: Kept this for now as a reference if we needed it in the future, but I could remove it completely.

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -6,10 +6,7 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - name: build-and-test
-      uses: agemooij/github-actions-sbt@master
```
Comment (maasg) [inline .github/workflows/push.yml]: will this work without the docker image specified here? Or should we use another?

File: core/build.sbt [hunk 3]
```diff
@@ -276,9 +276,6 @@ lazy val plugin =
 
       buildInfoKeys := Seq[BuildInfoKey](version),
       buildInfoPackage := "cloudflow.sbt",
-
-      resolvers += Resolver.url("lightbend-oss", url("https://lightbend.bintray.com/cloudflow"))(Resolver.ivyStylePatterns),
```
Comment (maasg) [inline core/build.sbt]: I guess that we need to wait for Sonatype to test this, right?

File: .github/workflows/push.yml (hunk 2 above)
Comment (skonto) [reply .github/workflows/push.yml]: I run the script locally it did work.

File: core/build.sbt (hunk 3 above)
Comment (skonto) [reply core/build.sbt]: I didnt see any issues building things locally.

**Comment by skonto** [comment]
@maasg I replied to the comments.

File: core/build.sbt (hunk 3 above)
Comment (debasishg) [reply core/build.sbt]: I removed this in my PR https://github.com/lightbend/cloudflow/pull/9 .. @skonto did u test removing artifacts from your local repo ? Just to check if it's being actually pulled from remote ?

Comment (skonto) [reply core/build.sbt]: yes I removed everything from both maven and ivy.

**Comment by skonto** [comment]
@debasishg rebased my PR and tried to build an example:
```
call-record-aggregator$ sbt build
[info] Loading settings for project call-record-aggregator-build from plugins.sbt,cloudflow-plugins.sbt ...
[info] Loading project definition from /home/stavros/Desktop/workspace/Lightbend/cloudflow/oss/post_launch/cloudflow/examples/call-record-aggregator/project
[info] Updating ProjectRef(uri("file:/home/stavros/Desktop/workspace/Lightbend/cloudflow/oss/post_launch/cloudflow/examples/call-record-aggregator/project/"), "call-record-aggregator-build")...
[info] downloading https://repo1.maven.org/maven2/org/scalariform/scalariform_2.12/0.2.5/scalariform_2.12-0.2.5.jar ...
[info] downloading https://repo1.maven.org/maven2/org/scala-lang/modules/scala-xml_2.12/1.0.6/scala-xml_2.12-1.0.6.jar ...
[info] downloading https://repo1.maven.org/maven2/org/scala-lang/scala-library/2.12.7/scala-library-2.12.7.jar ...
[info] downloading https://repo1.maven.org/maven2/com/lightbend/cloudflow/cloudflow-streamlets_2.12/1.3.0-M1/cloudflow-streamlets_2.12-1.3.0-M1.jar ...
[info] downloading https://repo1.maven.org/maven2/org/scalariform/sbt-scalariform_2.12_1.0/1.8.2/sbt-scalariform-1.8.2.jar ...
[info] downloading https://repo1.maven.org/maven2/com/lightbend/cloudflow/sbt-cloudflow_2.12_1.0/1.3.0-M1/sbt-cloudflow-1.3.0-M1.jar ...
...
```
It downloads from maven central so we are ok.

File: core/build.sbt (hunk 3 above)
Comment (debasishg) [reply core/build.sbt]: 👍

File: .github/workflows/push.yml (hunk 2 above)
Comment (maasg) [reply .github/workflows/push.yml]: `agemooij/github-actions-sbt` contains a `Dockerfile` that provides the environment where this action will run (see https://github.com/agemooij/github-actions-sbt/blob/master/Dockerfile)
I would expect that the action won't know how to do `sbt` without that image.

Comment (skonto) [reply .github/workflows/push.yml]: Ok I will setup sbt in there.

Comment (skonto) [reply .github/workflows/push.yml]: @maasg done.

File: .github/workflows/setup/Makefile [hunk 4]
```diff
@@ -0,0 +1,8 @@
+all: setup-local-all
```
Comment (skonto) [inline .github/workflows/setup/Makefile]: In the future we expand this file.

**Pull request merged**